src/libstd/sys/unix/fs.rs

```diff
@@ -459,9 +459,18 @@ pub fn rmdir(p: &Path) -> io::Result<()> {
 }
 
 pub fn readlink(p: &Path) -> io::Result<PathBuf> {
+    #[cfg(not(target_libc = "newlib"))]
```

Comment:
It comes from [`le32-unknown-nacl.mk`](https://github.com/DiamondLovesYou/rust/blob/master/mk/cfg/le32-unknown-nacl.mk), which I don't plan on upstreaming until the end.